templates/chart.twig

```diff
@@ -211,44 +211,74 @@
 
     var visJSON = {{ visualization | json | raw }};
 
-
-    __dw.init($.extend({
-        chartJSON: {{ chart.toStruct(true) | json | raw }},
-        preview: {{ preview ? 'true' : 'false' }},
-        chartLocale: '{{ chartLocale }}',
-        themeId: '{{ theme.id }}',
-        visId: '{{ visualization.id }}',
-        visJSON: visJSON,
-        metricPrefix: {{ metricPrefix | json | raw }},
-        lang: '{{ lang }}',
-        data: {{ chartData | json | raw }}
-    }, window.__dwParams));
-
-    if (/iP(hone|od|ad)/.test(navigator.platform)) {
-        window.onload = __dw.render();
-    }
-
-    if (visJSON.height == "fixed") {
-        setInterval(function() {
-            var desiredHeight = $('html').outerHeight(true);
-            window.parent.postMessage({
-                'datawrapper-height': {
-                    '{{ chart.id }}': desiredHeight
-                }
-            }, "*");
-
-            window.parent.postMessage({
-                sentinel: 'amp',
-                type: 'embed-size',
-                height: desiredHeight
-            }, '*');
-        }, 1000);
+    // load polyfills
+    (function() {
+        // see https://github.com/datawrapper/polyfills/blob/master/getBrowser.js
+        function getBrowser(){var e=!!window.opr&&!!opr.addons||!!window.opera||navigator.userAgent.indexOf(" OPR/")>=0;var r=typeof InstallTrigger!=="undefined";var o=/constructor/i.test(window.HTMLElement)||function(e){return e.toString()==="[object SafariRemoteNotification]"}(!window["safari"]||safari.pushNotification);var i=false||!!document.documentMode;var n=!i&&!!window.StyleMedia;var a=!!window.chrome&&!!window.chrome.loadTimes;var t=(a||e)&&!!window.CSS;var s=a?"chrome":r?"firefox":o?"safari":i?"ie":n?"edge":false;var f=a?getVersion(/chrom(?:e|ium)\/([0-9]+)\./):r?getVersion(/firefox\/([0-9]+\.*[0-9]*)/):o?getVersion(/version\/([0-9+]).[0-9+].[0-9+] safari/):i?getVersion(/msie ([0-9+]).[0-9+].[0-9+]/):n?getVersion(/edge\/([0-9+]).[0-9+].[0-9+]/):false;return{browser:s,version:f};function getVersion(e){var r=navigator.userAgent.toLowerCase().match(e);return r?parseInt(r[1],10):false}}
```

Comment:
good point. for now I will leave it like this, but as soon as we turn this code into ES2015+ with rollup this will be imported from the `@darawrapper/polyfills` package